devicon.json

```diff
@@ -7027,6 +7027,31 @@
             }
         ]
     },
+    {
+        "name": "splunk",
+        "altnames": [],
+        "tags": [
+            "platform",
+            "data",
+            "log",
+            "monitoring"
+        ],
+        "versions": {
+            "svg": [
+                "original-wordmark"
+            ],
+            "font": [
+                "original-wordmark"
+            ]
+        },
+        "color": "#010101",
+        "aliases": [
+            {
+                "base": "original-wordmark",
+                "alias": "plain-wordmark"
+            }
+        ]
```

Comment:
```suggestion
        ],
        "color": "#000"
```

Reply:
Ehh, this suggestion doesn't look right. Why would you put color inside of the alias object?

Reply:
hmm, did you want to change the color from `#010101` to `#000000` ?

Reply:
Looks like that's the case yeah.

@Panquesito7 can you confirm?

Reply:
Replaced the color with #000 in https://github.com/devicons/devicon/pull/1433/commits/49d18c9c00d2422a0182e3e52a6ad870cf95023e

Reply:
Wait a minute. The color should definitely be what it was earlier 👉 `#010101`
But maybe you could also change the color in the svg from a rgb() value to a hex value to make it more clear?

Reply:
> Ehh, this suggestion doesn't look right. Why would you put color inside of the alias object?

Hmm, it's not inside the `alias` array. 🤔
I'll create another comment for this. Thanks anyway. 🙂